eyelid-match-ops/benches/match-ops.rs

```diff
@@ -213,3 +222,23 @@ pub fn bench_mod_poly_ark(settings: &mut Criterion) {
         },
     );
 }
+
+/// Run [`poly::karatsuba_mul()`] as a Criterion benchmark with random data.
+pub fn bench_inv(settings: &mut Criterion) {
+    // Setup: generate random cyclotomic polynomials
+    let p = sample::<FULL_RES_POLY_DEGREE>();
```

Comment:
There might be some variability of timing introduced from sampling. Ideally this benchmark would use a deterministic RNG with a fixed seed.

It’s common for functions like `sample` to take `rng` as a parameter, and let each test decide which RNG to use.

Reply:
Done